python_ta/reporters/node_printers.py

```diff
@@ -89,13 +89,25 @@ def render_too_many_arguments(msg, source_lines=None):
     yield from render_generic(msg, source_lines)
 
 
+def render_missing_space_in_doctest(msg, source_lines=None):
+    """Render a missing space in doctest message"""
+    node = msg.node
+    line = msg.line
+    start = node.lineno
+
+    # Display code snippet from the start of the function to the line containing the error
+    yield from render_context(start, line, source_lines)
```

Comment:
For long docstrings it's not good to show this many lines. Instead, you should use the same general approach (in `render_generic`), which displays 2 lines before and after the erroneous line.